eq/fabric/client.cpp

```diff
@@ -54,31 +54,30 @@ bool Client::connectServer( co::NodePtr server )
     if( server->isConnected( ))
         return true;
 
-    co::ConnectionDescriptionPtr connDesc;
-    if( server->getConnectionDescriptions().empty( ))
-    {
-        connDesc = new co::ConnectionDescription;
-        connDesc->port = EQ_DEFAULT_PORT;
```

Comment:
I removed it without remembering there was a tool to start a server process.
I've never used the resident server process and I don't think I'll ever use it. For debugging purposes I've always found misleading to have a connection error trying to access the default server (where default means a different port in each machine because the uid may change, so it's not really hardcoded). Since having a resident server is not the primary use case, don't you think it's better to require the server URI explicitly in this case with EQ_SERVER (explicit better than implicit).